tests/Test/Xrefcheck/LocalSpec.hs

```diff
@@ -7,19 +7,20 @@ module Test.Xrefcheck.LocalSpec where
 
 import Universum
 
-import Test.Hspec (Spec, describe, it)
-import Test.QuickCheck ((===))
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.HUnit (testCase, (@?=))
 
 import Xrefcheck.Core (canonizeLocalRef)
 
-spec :: Spec
-spec = do
-  describe "Local refs canonizing" $ do
-    it "Strips ./" $
-      canonizeLocalRef "./AnchorsSpec.hs" === "AnchorsSpec.hs"
+test_local_refs_canonizing :: TestTree
+test_local_refs_canonizing = do
+  testGroup "Local refs canonizing" $ [
```

Comment:
Instead of: 

```hs
  testGroup "Local refs canonizing" $ [
    testCase "Strips ./" $
      undefined

    , testCase "Strips ././" $
      undefined
    ]
```

Can you please format lists like the example [shown here](https://github.com/serokell/style/blob/master/haskell.md#list-declarations)?

```hs
  testGroup "Local refs canonizing"
    [ testCase "Strips ./" $
        testBody

    , testCase "Strips ././" $
        testBody
    ]
```

I.e., the `[`, `,` and `]` are all aligned, and `testBody` has a 2-space indentation relative to `testCase`

Reply:
Also, I just realized the link in the PR template to the code style doc is broken, I fixed it in #154

Reply:
Done (I hope that everywhere)